endpoint/endpoint.go

```diff
@@ -393,7 +393,10 @@ func (e *Endpoint) UniqueOrderedTargets() {
 func FilterEndpointsByOwnerID(ownerID string, eps []*Endpoint) []*Endpoint {
 	filtered := []*Endpoint{}
 	for _, ep := range eps {
-		if endpointOwner, ok := ep.Labels[OwnerLabelKey]; !ok || endpointOwner != ownerID {
+		endpointOwner, ok := ep.Labels[OwnerLabelKey]
+		if !ok {
+			log.Debugf(`Skipping endpoint %v because of missing owner label (required: "%s")`, ep, ownerID)
```

Comment:
would be nice if both messages follow similar pattern either `(required: "%s")` or `required: "%s"`